tests/e2e/config/multicluster.json

```diff
@@ -0,0 +1,14 @@
+[
+    {
+      "cluster_name": "primary",
+      "pod_subnet": "10.10.0.0/16",
+      "svc_subnet": "10.255.10.0/24",
+      "network_id": "0"
+    },
+    {
+      "cluster_name": "remote",
+      "pod_subnet": "10.20.0.0/16",
+      "svc_subnet": "10.255.20.0/24",
+      "network_id": "0"
+    }
```

Comment:
Ok. I take it the multicluster setup from Istio upstream; I'll test with different networks because it made more sense to do it like that to simulate a more real scenario